content/docs/command-reference/config.md

```diff
@@ -63,7 +63,7 @@ multiple projects or users, respectively.
 > Note that the `--show-origin` flag can show you where a given config option
 > `value` is currently stored.
 
-## Command options (flags)
+## Command options/flags
```

Comment:
Renamed here and in `remote add/modify` to disambiguate vs. the `option` argument (remote/config parameters).